modules/core/src/loaders/xviz-loader-interface.js

```diff
@@ -197,6 +199,16 @@ export default class XVIZLoaderInterface {
     }
   );
 
+  getStreamMetadata = getXVIZConfig().DYNAMIC_STREAM_METADATA
```

Comment:
Same comment as `streamMetadata -> streamsMetadata`, I'd rename this `getStreamMetadata -> getStreamsMetadata`